eks-ebs-enable/README.md

```diff
@@ -0,0 +1,169 @@
+
+> NOTE: this tool only runs against standard AWS regions (and not US GovCloud or China regions)
+
+## Requirements
+
+You will need specific permissions to enable the addon. These permissions are currently outside the recommended set of permissions for Rancher.
+
+There are 2 options to supply these permissions
+
+### Option 1 - augment the permissions used when provisioning the cluster
+
+The IAM user who's credentials where used to provision the cluster can be augumented by adding the following permissions (via a policy). To do this with the `aws` cli you can run the following command to create the policy:
+
+```shell
+aws iam create-policy --policy-name Rancher_Support_EKSEBS --policy-document `
+`{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "eks:DescribeAddonConfiguration",
+                "eks:UpdateAddon",
+                "eks:ListAddons",
+                "iam:CreateOpenIDConnectProvider",
+                "eks:DescribeAddon",
+                "iam:ListOpenIDConnectProviders",
+                "eks:DescribeIdentityProviderConfig",
+                "eks:DeleteAddon",
+                "eks:CreateAddon",
+                "eks:DescribeAddonVersions",
+                "sts:AssumeRoleWithWebIdentity",
+                "eks:AssociateIdentityProviderConfig",
+                "eks:ListIdentityProviderConfigs"
+            ],
+            "Resource": "*"
+        }
+    ]
+}`
+
+```
+
+You can then attach this policy to the IAM user of the AWS credentials used by Rancher:
+
+```shell
+aws iam attach-user-policy --user-name <USERNAME> --policy-arn "arn:aws:iam::<ACCOUNT_ID>:policy/Rancher_Support_EKSEBS
+```
+
+You will need to replace `<USERNAME>` with the IAM user that the Rancher credentials belong to and also `<ACCOUNT_ID>` with your AWS account id.
+
+### Option 2 - permissions just for this tool
+
+Create a new IAM policy with the following permissions:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "iam:GetRole",
+                "eks:DescribeAddonConfiguration",
+                "eks:UpdateAddon",
+                "eks:ListAddons",
+                "iam:CreateRole",
+                "iam:AttachRolePolicy",
+                "eks:DescribeAddon",
+                "iam:CreateOpenIDConnectProvider",
+                "iam:PassRole",
+                "eks:DescribeIdentityProviderConfig",
+                "eks:DeleteAddon",
+                "iam:ListOpenIDConnectProviders",
+                "iam:ListAttachedRolePolicies",
+                "eks:CreateAddon",
+                "eks:DescribeCluster",
+                "eks:DescribeAddonVersions",
+                "sts:AssumeRoleWithWebIdentity",
+                "eks:AssociateIdentityProviderConfig",
+                "eks:ListIdentityProviderConfigs"
+            ],
+            "Resource": "*"
```

Comment:
```suggestion
            "Resource": [
                "*"
                ]
```

Reply:
Not changed, the current version is less verbose